src/sync.py

```diff
@@ -0,0 +1,27 @@
+from src.jira import JiraIssue
+
+
+class Sync:
```

Comment:
This seems to be used for demo. If true, can you add it to the demo module you created and perhaps just write one sentence of explanation ("Purpose is ....")? If not, I would take out the hard-coded repos.

Reply:
removed